source/to_cpp1.h

```diff
@@ -1245,8 +1254,11 @@ class cppfront
 
         //  Now we'll open the Cpp1 file
         auto cpp1_filename = sourcefile.substr(0, std::ssize(sourcefile) - 1);
-        if (!flag_cpp1_filename.empty()) {
-            cpp1_filename = flag_cpp1_filename; // use override if present
+        if (!flag_cpp1_filename.empty()) { // use override if present
+            cpp1_filename = flag_cpp1_filename;
+        }
+        else if (!flag_cwd.empty()) {      // strip leading path if cwd was explicitly set
```

Comment:
If we make this like be just `else {`, then I think the semantics would be that you get your desired behavior if `-cwd` is specified, _and_ we get the desired behavior change of putting the output in the current directory, correct?